<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fso::fred::dialogs::WingEditorDialog</class>
 <widget class="QDialog" name="fso::fred::dialogs::WingEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>551</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>443</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>443</width>
    <height>551</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wings Editor</string>
  </property>
  <widget class="QComboBox" name="selectWingCombobox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select Wing</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="generalGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>421</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>General</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="wingNameLabel">
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Wing Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="wingNameLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="totalWavesLabel">
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Total Waves</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="totalWavesSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="hotkeyLabel">
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Hotkey</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="hotkeyComboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="wingLeaderLabel">
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Wing Leader</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="wingLeaderComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="waveThresholdLabel">
         <property name="text">
          <string>Wave Threshold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="waveThresholdSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="squadLogoLabel">
         <property name="text">
          <string>Squad Logo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="squadLogoLineEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="browseSquadLogosButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="toggleGeneralOptionsButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="previousWingButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Prev Wing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextWingButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Wing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disbandWingButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Disband</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteWingButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="initialOrdersButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Initial Orders</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="wingFlagsGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Wing Flags</string>
   </property>
   <widget class="QToolButton" name="toggleWingFlagsButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="reinforcementUnitFlagCheckbox">
       <property name="text">
        <string>Reinforcement Unit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ingoreCountingGoalsFlagCheckbox">
       <property name="text">
        <string>Ignore for Counting Goals</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="noArrivalMusicFlagCheckbox">
       <property name="text">
        <string>No Arrival Music</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>191</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QCheckBox" name="noArrivalMessageFlagCheckbox">
       <property name="text">
        <string>No Arrival Message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="noFirstWaveMessageCheckbox">
       <property name="text">
        <string>No First Wave Message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="noDynamicGoalsCheckbox">
       <property name="text">
        <string>No Dynamic Goals</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTabWidget" name="arrivalDeparturesTabs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>421</width>
     <height>201</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="arrivalLocationTab">
    <attribute name="title">
     <string>Locations and Delays</string>
    </attribute>
    <widget class="QLabel" name="arrivalTypeLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Arrival Type</string>
     </property>
    </widget>
    <widget class="QComboBox" name="arrivalTypeComboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Warp In at Current Location</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Start in Docking Bay</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Warp Near Ship</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Warp In Front of Ship</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Warp In Back of Ship</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Warp In Left of Ship</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Warp In Right of Ship</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Warp In Above Ship</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Warp In Below Ship</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="departureTypeLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Departure Type</string>
     </property>
    </widget>
    <widget class="QComboBox" name="departureTypeComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Warp In</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Docking Bay</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="arrivalTargetComboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="initialArrivalDelayLabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Initial Arrival Delay</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="initialArrivalDelaySpinBox">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="arrivalTargetLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Target</string>
     </property>
    </widget>
    <widget class="QLabel" name="arrivalDistanceLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Distance</string>
     </property>
    </widget>
    <widget class="QLabel" name="preDepartureDelayLabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pre-Departure Delay</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="preDepartureDelaySpinBox">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>110</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="arrivalDistanceLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="delayBetweenWavesGroupBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Delay Between Waves</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>188</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="minWaveDelayLabel">
         <property name="text">
          <string>Min</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="minWaveDelayLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="maxWaveDelayLabel">
         <property name="text">
          <string>Max</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maxWaveDelayLineEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QComboBox" name="departureTargetComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="departureTargetLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Target</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="conditionalsTab">
    <attribute name="title">
     <string>Conditionals</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="pararmetersTab">
    <attribute name="title">
     <string>Parameters</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="arrivalDepartureLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Arrival and Departure</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toggleArrivalDepartureButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>^</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
